api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -969,6 +1000,18 @@ public Concept getNonCodedDrugConcept() {
 		return null;
 	}
 	
+	@Override
+	@Transactional(readOnly = true)
+	public OrderGroup getOrderGroupByUuid(String uuid) throws APIException {
+		return dao.getOrderGroupByUuid(uuid);
+	}
+	
+	@Override
+	@Transactional(readOnly = true)
+	public OrderGroup getOrderGroupById(Integer orderGroupId) throws APIException {
```

Comment:
rename to getOrderGroup(Integer orderGroupId) to be consistent with other services.